libenkf/src/enkf_state.c

```diff
@@ -653,12 +653,24 @@ static void enkf_state_internalize_GEN_DATA(enkf_state_type * enkf_state ,
                                             const model_config_type * model_config ,
                                             int last_report) {
 
-  member_config_type * my_config     = enkf_state->my_config;
-  const int  iens                    = member_config_get_iens( my_config );
   stringlist_type * keylist_GEN_DATA = ensemble_config_alloc_keylist_from_impl_type(enkf_state->ensemble_config,
                                                                                     GEN_DATA);
+
+
+  if (stringlist_get_size( keylist_GEN_DATA) > 0) {
+    if (last_report <= 0) {
```

Comment:
Should this be `last_report < 0`? I think it would increase readability and decrease "nestedness" if the two if statements were merged into one by an and.